stl/inc/__msvc_int128.hpp

```diff
@@ -1332,6 +1357,5 @@ _STD_END
 _STL_RESTORE_CLANG_WARNINGS

```

Comment:
`_TEMPLATE_CLASS_INTEGRAL` is not used elsewhere. Perhaps it should be `#undef`ed at the end of `__msvc_int128.hpp`.